docs/rfcs/notification-component.md

```diff
@@ -0,0 +1,90 @@
+# Notification Component
+
+Notifications are inline messages that are part of the content of a UI, and alert users to a specific screen area that requires their action or attention.  They may appear within a page or dialog.  They are not dismissable, and only disappear once the issue has been resolved.
+
+## Usage
+
+### Basic Usage
+
+```javascript
+import React from 'react';
+import Notification from 'mineral-ui/Notification';
+
+export default () => {
+  return (
+    <Notification
+      title="Weekly Report Available"
+      variant="info"
+      actions={
+        text: 'View Report'
+      }>
+      <Text>
+        Your screen time was down 7% last week, for an average of 2 hours, 53 minutes a day.
+      </Text>
+    </Notification>
+  );
+}
+```
+
+## Exports
+
+* `Notification` (default)
+
+## API
+
+### `Notification` API
+
+| Prop | Type | Required | Default | Description |
+| ---- | ---- | -------- | ------- | ----------- |
+| actions | Array<{}> | false | | Array of Button props |
+| children | React$Node | false | | Body content of the Notification |
+| compact | boolean | false | | Place actions on same line as title. Disallows body content - title only. |
+| title | string \| React$Node | true | | Title of the Notification |
+| variant | 'info' \| 'danger' \| 'success' \| 'warning' | false | 'info' | Intent of the Notification |
+| ...rest | any | false | | All other props will be applied to the root element |
+
+## Dependencies
+
+* `Button` component used for actions
+* Not a direct dependency, but it is expected that any content links are built using mineral-ui `Link` component
```

Comment:
I think, because of the need to _not_ focus notifications (see comment below), the body content cannot contain interactive elements.

Unless, of course, we provide some kind of focus management.

Reply:
Gonna put this conversation on hold until we ensure that everyone is using the proper nomenclature for the different messaging components.